ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -167,6 +171,11 @@ export class ObjectDetailComponent implements OnInit {
     this.localTableData = JSON.parse(JSON.stringify(this.tableData))
     this.localIndexData = JSON.parse(JSON.stringify(this.indexData))
 
+    if(this.srcDbName == SourceDbNames.MySQL && this.spColspan<7) {
+      this.spDisplayedColumns.splice(2, 0, "spAutoGen")
+      this.spColspan++
+    }
```

Comment:
what is `this.spColspan < 7`?

Reply:
Its so that when reredering, it doesn't keep adding the column. Modified the condition to just check for the value in the array. I forgot this was easy in js unlike golang 😅